src/Router.tsx

```diff
@@ -1,110 +1,95 @@
 import { EnrollmentView } from "./views/login/EnrollmentView";
-import { AdminHomeView } from "./views/AdminHomeView";
-import { UserManagementView } from "./views/UserManagementView";
-import { UserInviteView } from "./views/UserInviteView";
-import { QrCodeView } from "./views/users/invite/QrCodeView";
-import { EnrollCodeListView } from "./views/users/invite/EnrollCodeListView";
-import { EnrollApprovalView } from "./views/users/invite/EnrollApprovalView";
-import { ServicesView } from "./views/ServicesView";
-import { ServiceTakView } from "./views/ServiceTakView";
-import { SoldierServicesView } from "./views/SoldierServicesView";
-import { SoldierServiceTakView } from "./views/SoldierServiceTakView";
-import { MtlsTestView } from "./views/MtlsTestView";
 import { ErrorView } from "./views/ErrorView";
+import { NotFoundView } from "./views/NotFoundView";
+import { TakRoutes } from "./TakRoutes";
+import { MtlsCreateView } from "./views/mtls/MtlsCreateView";
+import { AdminRouteWrapper } from "./AdminRouteWrapper";
+import { UserRouteWrapper } from "./UserRouteWrapper";
+import { TakRouteWrapper } from "./TakRouteWrapper";
 
 const router = createBrowserRouter([
   {
     path: "/",
-    children: [
-      {
-        path: "/login",
-        element: <LoginView />,
-      },
-      {
-        path: "/login/callsign",
-        element: <CallsignSetupStep />,
-      },
-      {
-        path: "/login/enrollment",
-        element: <EnrollmentView />,
-      },
-      {
-        path: "/app",
-        element: <AdminHomeView />,
-      },
-      {
-        path: "/app/user-management",
-        element: <UserManagementView />,
-      },
-      {
-        path: "/app/user-management/invite",
-        element: <UserInviteView />,
-      },
-      {
-        path: "/app/users/:callsign",
-        element: <SoldierView />,
-      },
-      {
-        path: "/app/user-management/code-list",
-        element: <EnrollCodeListView />,
-      },
-      {
-        path: "/app/user-management/code-list/:inviteCode",
-        element: <QrCodeView />,
-      },
-      {
-        path: "/app/user-management/approval",
-        element: <EnrollApprovalView />,
-      },
-      {
-        path: "/enrollment",
-        element: <MassEnrollmentView />,
-      },
-      {
-        path: "/app/services",
-        element: <ServicesView />,
-      },
-      {
-        path: "/app/services/tak",
-        element: <ServiceTakView />,
-      },
-      {
-        path: "/app/users/:callsign/services/tak",
-        element: <SoldierServiceTakView />,
-      },
-      {
-        path: "/app/users/:callsign/services",
-        element: <SoldierServicesView />,
-      },
-      {
-        path: "/app/mtls-test",
-        element: <MtlsTestView />,
-      },
-      {
-        path: "/error",
-        element: <ErrorView />,
-      },
-    ],
+    element: (
+      <ProtectedRoute>
+        <></>
+      </ProtectedRoute>
```

Comment:
Why not just <ProtectedRoute /> ?

Reply:
If so, gives this typescript error:
[{
	"resource": "/Users/bg/PVARKI-projekti/esitesivu/docker-rasenmaeher-integration/ui/src/Router.tsx",
	"owner": "typescript",
	"code": "2741",
	"severity": 8,
	"message": "Property 'children' is missing in type '{}' but required in type 'Props'.",
	"source": "ts",
	"startLineNumber": 18,
	"startColumn": 8,
	"endLineNumber": 18,
	"endColumn": 22,
	"relatedInformation": [
		{
			"startLineNumber": 6,
			"startColumn": 3,
			"endLineNumber": 6,
			"endColumn": 11,
			"message": "'children' is declared here.",
			"resource": "/Users/bg/PVARKI-projekti/esitesivu/docker-rasenmaeher-integration/ui/src/hook/auth/ProtectedRoute.tsx"
		}
	]
}]